grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -88,6 +88,12 @@ class AuthTagLib {
             out << body()
     }
 
+    def userHasRoles = {attrs, body ->
```

Comment:
Again, use a boolean verb here.

Reply:
Actually this should probably be ifUserHasRoles since that's the behavior we seem to be implementing with this tag, right?

Reply:
yes